ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/monitor-layer1.j2

```diff
@@ -11,8 +11,13 @@
 | Layer 1 Event | Logging |
 | ------------- | ------- |
 | MAC fault | {{ monitor_layer1.logging_mac_fault | arista.avd.default(false) }} |
-| Transceiver DOM | {{ monitor_layer1.logging_transceiver.dom | arista.avd.default(false) }} |
-| Transceiver communication | {{ monitor_layer1.logging_transceiver.communication | arista.avd.default(false) }} |
+{%     if monitor_layer1.logging_transceiver.enabled is arista.avd.defined(true) %}
+| Logging Transceiver | {{ monitor_layer1.logging_transceiver.enabled }} |
+{%     endif %}
+{%     if monitor_layer1.logging_transceiver.dom is arista.avd.defined or monitor_layer1.communication is arista.avd.defined %}
+| Transceiver DOM | {{ monitor_layer1.logging_transceiver.dom }} |
+| Transceiver communication | {{ monitor_layer1.logging_transceiver.communication }} |
+{%     endif %}
```

Comment:
It is not necessary that both of them have to enable at the same time
```suggestion
{%     if monitor_layer1.logging_transceiver.dom is arista.avd.defined %}
| Transceiver DOM | {{ monitor_layer1.logging_transceiver.dom }} |
{%.    endif %}
{%     if monitor_layer1.communication is arista.avd.defined%}
| Transceiver communication | {{ monitor_layer1.logging_transceiver.communication }} |
{%     endif %}
```

Reply:
It is checking if any one of them is defined.

Reply:
Fixed.